reactivesocket-cpp/src/Frame.h

```diff
@@ -5,15 +5,15 @@
 #include <iosfwd>
 #include <limits>
 
-#include <folly/Optional.h>
+/// Needed for inline d'tors of frames.
+#include <folly/io/IOBuf.h>
+#include <folly/io/IOBufQueue.h>
```

Comment:
the two includes above seem to be necessary for the gcc build, otherwise resulting in [these](https://travis-ci.org/ReactiveSocket/reactivesocket-cpp/jobs/142580658) errors